File: app/src/main/java/com/lkms/data/model/java/UserManual.java
package com.lkms.data.model.java;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

/**
 * Đại diện cho một hàng trong bảng "UserManual".
 */
@Data
public class UserManual {

    @SerializedName("manualId")
    private String manualId;

    @SerializedName("url")
    private String url;
}

File: app/src/main/java/com/lkms/data/model/UserManual.kt
// Trong file: data/model/UserManual.kt
package com.lkms.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * Đại diện cho một hàng trong bảng "UserManual".
 */
@Serializable
data class UserManual(
    @SerialName("manualId")
    val manualId: String? = null,

    @SerialName("url")
    val url: String? = null
)